atlp-rwanda/stackup2-barefoot-frontend
# #171380683 setup Redux React-Router-dom

#### What does this PR do?
- setup redux for managing state
- setup react router dom for routing in our app

#### Description of Task to be completed?
- installed redux package
- installed react-redux package
- created actions folder
- created store folder
- created app central store 
- connected the app with the store
- installed react-router-dom package
- created the views folder which contains the presentational components

#### How should this be manually tested?
- Clone the project repo
- Run `cd stackup2-barefoot-frontend`
- Rename `.env.example` to `.env` and provide all the required values
- Run the following commands respectively:
> `git fetch origin ch-setup-redux-react-router-dom-171380683`
> `git checkout ch-setup-redux-react-router-dom-171380683`
> `yarn install`
> `yarn start`
- on
- to check for redux click on `try redux` button then you will get `Redux is working !!!` message
- to check for react-router-dom click on `login` button to go to another page

#### Any background context you want to provide?
- you need to have node installed on your machine

#### What are the relevant pivotal tracker stories?
  [#171380683](https://www.pivotaltracker.com/story/show/171380683)
  [#171380686](https://www.pivotaltracker.com/story/show/171380686)

#### Screenshots (if appropriate)
N/A
